config/test_helpers.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
 )
 
-// LoadConfigFile loads a config from file.
+// LoadConfigFile loads a config from file, without verifying it.
 func LoadConfigFile(t *testing.T, fileName string, factories Factories) (*configmodels.Config, error) {
```

Comment:
This changes exported functionality that is used from Contrib repo. I suggest we keep `LoadConfigFile` behavior unchanged (i.e. add `ValidateConfig` call to it).

We probably don't need a version of this function that does not validate the config. I think it is only needed `TestLoadEmptyConfig` so just call `config.Load` directly.

Reply:
Done.